### ISSUE TITLE
Display LoD2 data in FE

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#162]
## Depends on
#161 
#

## Acceptance criteria
* Display 3d LoD2 model
* Keeping performance in mind

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by op-hub** [comment]
Performance wise, displaying lod2 data in FE is not feasible. See #78
